{{cookiecutter.project_slug}}/CONTRIBUTING.rst

```diff
@@ -70,16 +70,14 @@ Commit Changes
 How to
 ~~~~~~
 
-1. Fork the `{{ cookiecutter.project_slug }}` repo on GitLab.
-2. Clone your fork locally::
+1. Clone the repository::
 
     $ git clone {{ giturl }}
 
-3. Install your local copy into a virtualenv. Assuming you have virtualenvwrapper installed, this is how you set up your fork for local development::
+2. Create an environment::
```

Comment:
We are jumping from 2 to 4. Should not we also install the package using `pip install .`? That should be the replacement for `python setup.py install`.